src/App.tsx

```diff
@@ -17,6 +17,7 @@ const App = () => {
 					<Route path="/login" element={<Login />} />
 					<Route path="/signup" element={<Signup />} />
 					<Route path="/verify" element={<Verification />} />
+					<Route path="*" element={<NotFound />} />
```

Comment:
You need to edit the route so that `NotFound` page get showed on `/404` route because
currently `NotFound` is shown on `/hi` route

```jsx
import { Navigate } from 'react-router-dom'

<Route path="/404" element={<NotFound />} />
<Route path="*" element={<Navigate replace to="/404" />} />
```